src/frontend/src/tests/sol/derived/tokens.derived.spec.ts

```diff
@@ -0,0 +1,56 @@
+import * as solEnv from '$env/networks/networks.sol.env';
+import {
+	SOLANA_DEVNET_TOKEN,
+	SOLANA_LOCAL_TOKEN,
+	SOLANA_TESTNET_TOKEN,
+	SOLANA_TOKEN
+} from '$env/tokens/tokens.sol.env';
+import * as appContants from '$lib/constants/app.constants';
+import { testnetsStore } from '$lib/stores/settings.store';
+import { enabledSolanaTokens } from '$sol/derived/tokens.derived';
+import { get } from 'svelte/store';
+
+describe('tokens.derived', () => {
+	describe('enabledSolanaTokens', () => {
+		beforeEach(() => {
+			vi.resetAllMocks();
+
+			testnetsStore.reset({ key: 'testnets' });
+
+			vi.spyOn(solEnv, 'SOLANA_NETWORK_ENABLED', 'get').mockImplementation(() => true);
+
+			vi.spyOn(appContants, 'LOCAL', 'get').mockImplementation(() => false);
+		});
+
+		it('should return only mainnet token by default', () => {
+			expect(get(enabledSolanaTokens)).toEqual([SOLANA_TOKEN]);
+		});
+
+		it('should return emtpy array if feature flag false', () => {
+			vi.spyOn(solEnv, 'SOLANA_NETWORK_ENABLED', 'get').mockImplementation(() => false);
+			expect(get(enabledSolanaTokens)).toEqual([]);
+		});
+
+		it('should return testnet tokens when they are enabled', () => {
+			testnetsStore.set({ key: 'testnets', value: { enabled: true } });
+
+			expect(get(enabledSolanaTokens)).toEqual([
+				SOLANA_TOKEN,
+				SOLANA_TESTNET_TOKEN,
+				SOLANA_DEVNET_TOKEN
+			]);
+		});
+
+		it('should return localnet token when in local env', () => {
```

Comment:
Also, this test doesn't make sense anymore since we removed the `LOCAL` flag usage from `testnet` store. We can remove this test suite completely since the previous one already validates the needed behavior.

Reply:
Hmm, I think they still make sense:
First case: If we are not local => return SOL, SOL testnet and SOL devnet token
Second case: If we are local => return SOL, SOL testnet, SOL devnet and SOL local token

Reply:
Ah, true, didn't check the test properly